benchmark/src/main/java/org/corfudb/benchmark/CorfuTableBenchmark.java

```diff
@@ -0,0 +1,194 @@
+package org.corfudb.benchmark;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.reflect.TypeToken;
+import lombok.Data;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'lombok.RequiredArgsConstructor'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)